app/src/main/java/co/kr/sopt_seminar_30th/data/repositoryimpl/remote/HomeRepositoryImpl.kt

```diff
@@ -0,0 +1,62 @@
+package co.kr.sopt_seminar_30th.data.repositoryimpl.remote
+
+import co.kr.sopt_seminar_30th.data.service.home.HomeService
+import co.kr.sopt_seminar_30th.domain.entity.home.UserFollowInformation
+import co.kr.sopt_seminar_30th.domain.entity.home.UserProfileInformation
+import co.kr.sopt_seminar_30th.domain.entity.home.UserRepositoryInformation
+import co.kr.sopt_seminar_30th.domain.repository.remote.HomeRepository
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class HomeRepositoryImpl @Inject constructor(
+    private val service: HomeService,
+    private val coroutineDispatcher: CoroutineDispatcher
+) : HomeRepository {
+    override suspend fun fetchUserInformation(userId: String): Result<UserProfileInformation> {
```

Comment:
```suggestion
    override suspend fun fetchUserInformation(userId: String): Result<UserProfileInformation> = withContext(...
```

[reference](https://kotlinlang.org/docs/coding-conventions.html#functions)

Reply:
👍